.wordpress-org/blueprints/blueprint.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://playground.wordpress.net/blueprint-schema.json",
+  "landingPage": "\/wp-admin\/post.php?post=7&action=edit",
+  "preferredVersions": {
+    "php": "7.4",
+    "wp": "latest"
+  },
+  "phpExtensionBundles": [
+    "kitchen-sink"
+  ],
+  "features": {
+    "networking": true
+  },
+  "steps": [
+    {
+      "step": "login",
+      "username": "admin",
+      "password": "password"
+    },
+    {
+      "step": "installPlugin",
+      "pluginZipFile": {
+        "resource": "wordpress.org\/plugins",
+        "slug": "embed-block-figma"
+      },
+      "options": {
+        "activate": true
+      }
+    },
+    {
+      "step": "importFile",
+      "file": {
+        "resource": "url",
+        "url": "https:\/\/raw.githubusercontent.com\/10up\/figma-block\/05a11e7ed51038173610e543cb02fc56024f0c1f\/.wordpress-org\/blueprints\/demo-data.xml"
```

Comment:
We'll probably want to ensure this URL get's updated to point to `develop` before this is merged